src/tui/light.go

```diff
@@ -132,11 +121,10 @@ func NewLightRenderer(theme *ColorTheme, forceBlack bool, mouse bool, tabstop in
 	return &r
 }
 
-func (r *LightRenderer) fd() int {
-	return int(r.ttyin.Fd())
-}
-
 func (r *LightRenderer) defaultTheme() *ColorTheme {
+	if util.IsWindows() && os.Getenv("ConEmuANSI") == "ON" {
```

Comment:
winpty doesn't unset `ConEmuANSI` (#106) so this can break Vim `:terminal` on Windows.

Reply:
I'm fine with removing this check unless you have another suggestion.

Reply:
It's ok to keep it now. There's a way to modify or unset environment variables for (neo)vim terminal without relying on the shell.